### DISCUSSION TITLE
CameraX Survey: make your thoughts/desires heard :-)

### DISCUSSION THREAD

**Discussion by ggfan** [android/camera-samples#401] [Announcements]
Hello Awesome Android CameraX/Camera Developers:

We, CameraX team, like to hear your thoughts about CameraX (Camera2 too) API, may you please help "encode" your experiences/thoughts/wishes into [**this survey**](https://forms.gle/DarkBYwWFmDzdtLr8)? This is also the opportunity to influence the CameraX/Camera API future direction, do not miss it:  your experiences, thoughts, and wishes will be "captured and real-time decoded" by the team!

Again the survey is [**here**](https://forms.gle/DarkBYwWFmDzdtLr8). Thank you in advance!

**Comment by jimie-chaos** [comment]
Can you support OverlayEffect example?
